README.md

```diff
@@ -78,125 +78,140 @@ settings](config/babel/.babelrc).
 
 ```
 $ builder help builder-react-component
-[builder:help]
 
 Usage:
 
-  builder [action] [task]
+  builder builder-react-component [flags] [task]
 
 Actions:
 
   help, run, concurrent, install
 
+Flags:
+
+  --builderrc: Path to builder config file (default: `.builderrc`)
+
 Tasks:
 
+  builder:check
+    npm run builder:lint
+
+  builder:lint
+    npm run builder:lint-server
+
+  builder:lint-server
+    eslint --color -c config/eslint/.eslintrc-server config/karma config/webpack
+
+  builder:prepublish
+    node util/dev.js
+
   npm:postinstall
-    [builder-react-component] node -e "require('fs').stat('lib', function(e,s){process.exit(e || !s.isDirectory() ? 1 : 0)})" || builder run build-lib
```

Comment:
At the point that we start allowing "multiple" archetypes (if that happens), having the archetype in `[ ]` is a really great debugging tool.